example/common/global_example_menu.dart

```diff
@@ -1,6 +1,6 @@
-import 'package:over_react/over_react.dart';
 import 'dart:html';
 
+import 'package:over_react/components.dart' show ErrorBoundary;
```

Comment:
I fixed some deprecation warnings for `ErrorBoundary`/`setClientConfiguration` while I was in here